microprofile/server/src/main/java/io/helidon/microprofile/server/ServerCdiExtension.java

```diff
@@ -416,31 +448,40 @@ private static int priority(Class<?> aClass) {
     private void registerWebServerService(Class<?> serviceClass,
```

Comment:
Is this method actually called?  I see that it is referenced in `registerWebServerServices`, but that's `private` and I don't see _it_ being called anywhere.  Maybe I'm missing something?

Reply:
Sure it is, github manages to hide it but ist called from `startServer` method https://github.com/oracle/helidon/blob/320be29cc2c9779f53a10a8de7bd64e61fb6cfc4/microprofile/server/src/main/java/io/helidon/microprofile/server/ServerCdiExtension.java#L167-L211